Three fixes.

Y-Less opened the pull request:
One is an internal fix - making `FIXES_gscSpace` actually `const` in as many places as possible.  Specifically, all user-facing uses are `const` meaning that no-one can accidentally change the string.  The only functions that don't take a `const` string are only used within the library itself so the variable can be made `static` instead to prevent external modifications.  This means there are now two space strings, but that's not a major issue.

The second isn't really a "fix", but is in line with the existing `IS_FILTERSCRIPT` variable - just adds `IS_JIT`, `IS_WINDOWS` and `IS_LINUX` tests (should maybe also have `IS_GAMEMODE` and `IS_NPCMODE`, but currently don't, in fact I don't think this supports NPC modes at all).

The last one is to actually print in the console on Linux, instead of `print` only going to `server.log`.  It uses a trick from this topic:

http://forum.sa-mp.com/showthread.php?p=3812622#post3812622

Which requires an additional command to actually work on Linux:

```
ln -s /dev/tty ./scriptfiles/CON
```

Y-Less committed:
Better const-correctness on space strings.

Y-Less committed:
IS_JIT, IS_LINUX, and IS_WINDOWS.

Y-Less committed:
print and printf don't show in the console on Linux.

Requires an additional hack in the form of a symlink to connect `scriptfiles` to `stdout`:

```
    ln -s /dev/tty ./scriptfiles/CON
```

ziggi commented:
`print` problem can be easily fixed by adding `output 1` into the `server.cfg`. I think what this solution is much better than yours.

Y-Less commented:
I didn't know that was an option that existed!  That really is a better solution.

Y-Less committed:
Merge remote-tracking branch 'remotes/upstream/master'

Y-Less committed:
Fixed long line length.

Y-Less committed:
Revert "print and printf don't show in the console on Linux."

This reverts commit 6854a9d8091661791ccb0e4126734b66418b0726.

Y-Less committed:
Tried to correct newlines.

Y-Less commented:
Reverted the superfluous fix.  Corrected line endings in the GIT repository.  You can still check out with Windows line endings, but it should now commit with Unix endings and not show the horrible red ^M after every line in Github.

Y-Less commented:
Tagging @ziggi so you see I updated it.
